cli/tests/integration/cache_tests.rs

```diff
@@ -92,3 +92,8 @@ itest!(check_local_by_default2 {
   output: "cache/check_local_by_default2.out",
   http_server: true,
 });
+
+itest!(json_import {
+  args: "cache --quiet cache/json_import/main.ts",
+  output: "cache/json_import/main.out",
```

Comment:
Oh, I just realized this is empty. Wonder what I can do to avoid an empty file.

Reply:
I updated the test util to allow not providing anything.

Reply:
> Wonder what I can do to avoid an empty file.

`output_str: Some("")` but good addition anyway